vector/src/main/java/im/vector/app/features/userdirectory/UserListFragment.kt

```diff
@@ -86,7 +85,7 @@ class UserListFragment :
         setupRecyclerView()
         setupSearchView()
 
-        homeServerCapabilitiesViewModel.onEach {
+        viewModel.onEach {
```

Comment:
the  `isE2EByDefault` is not initialized correctly (except if you come from a new DM screen)